### DISCUSSION TITLE
Service Apache and MySQL auto run in windows service

### DISCUSSION THREAD

**Discussion by mohsetiawanmodeong** [leokhoa/laragon#680] [Q&A] [answered]
How to enable service apache and MySQL laragon auto run in windows service.
I want to auto run laragon and their service when windows a long run

**Comment by Pen-y-Fan** [comment, marked as answer]
See: Support for using windows services #627

**Reply by mohsetiawanmodeong** [reply]
i try install mysqld and apached to service.msc
